packages/web-runtime/src/store/config.js

```diff
@@ -39,9 +39,14 @@ const state = {
   },
   options: {
     defaultExtension: 'files',
-    homeFolder: '',
     disablePreviews: false,
     displayResourcesLazy: true,
+    homeFolder: '',
+    sidebar: {
+      shares: {
+        showAllOnLoad: false
```

Comment:
Please also add this to the getting started docs page, which lists all available options.